wasm/assembly/massa.ts

```diff
@@ -12,4 +15,32 @@ export function print(message: string): void {
 
 export function get_remaining_gas(): u64 {
     return get_remaining_points();
+}
+
+export function get(key: string): string {
+    return "";
+}
+
+export function set(key: string, value: string): void {
+
+}
+
+export function create_sc(bytecode: string): SmartContract {
+    return { address: "", bytecode };
+}
+
+export function get_bytecode(address: string): string {
+    return "";
```

Comment:
is it necessary to implement the function ? can't we just declare a prototype ?

Reply:
Yes it's necessary to implement this function (It will be a todo thing in a next issue :smile:) but you get the point, I wouldn't have to commit this file now!

Reply:
Oh ok I see ! No problem